src/components/Testimonials.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+
+import { getRandomElements, isMobile } from '@/lib/utils';
+
+import NextImage from '@/components/NextImage';
+
+import { Testimonial } from '@/constant/models';
+import { TESTIMONIAL_LOGOS, TESTIMONIALS } from '@/constant/testimonials';
+
+function TestimonialSectionAmberBox() {
+  return (
+    <div className='bg-primary-amber custom-angled-rectangle-amber-testimonials-animated-mobile lg:custom-angled-rectangle-amber-animated-testimonials absolute left-0 h-[700px] w-[80%] lg:w-[500px]' />
+  );
+}
+
+function TestimonialText({ testimonial }: { testimonial: Testimonial }) {
+  return (
+    <span className='font-primary text-left text-xs font-light lg:w-60'>
+      <span className='font-medium'>— {testimonial.source}</span>
+      <br />
+      <br />
+      <p>"{testimonial.text}"</p>
+    </span>
+  );
+}
+
+function TestimonialLogo({ imageSrc }: { imageSrc: string }) {
+  return (
+    <div className='flex h-24 w-60 shrink-0 items-center justify-center bg-white p-10 text-black'>
+      <NextImage
+        src={imageSrc}
+        alt='logo'
+        width='240'
+        height='90'
+        useSkeleton
+      />
+    </div>
+  );
+}
+
+export default function Testimonials() {
+  const [randomTestimonials, setRandomTestimonials] = useState<Testimonial[]>(
+    []
+  );
+  const [index, setIndex] = useState(0);
+  const [opacity, setOpacity] = useState<number>(1);
+
+  useEffect(() => {
+    const random: Testimonial[] = getRandomElements(TESTIMONIALS, 3);
+    setRandomTestimonials(random);
+
+    if (isMobile()) {
+      const changeText = () => {
+        setOpacity(0);
+        setTimeout(() => {
+          setIndex((prevIndex) => (prevIndex + 1) % TESTIMONIALS.length);
+          setOpacity(100);
+        }, 700); // fade duration
+      };
+
+      const timer = setInterval(changeText, 5000 + 700); // Time to show text + fade-out duration
```

Comment:
should be longer, its hard for me to read a testimonial fully before it switches

Reply:
How long would you make it? 7 seconds? 10 Seconds?

Reply:
The Ape Unit Statement took me about 12 seconds to read. I was only thinking not to use a too long timer because when you miss a statement you have to wait 1 minute to see it again. Maybe we should rethink if automated fading is the best solution 🤔

Reply:
Any alternative ideas, what to do instead of fading?